crates/bgp-pkt/src/wire/serializer/mod.rs

```diff
@@ -182,6 +182,37 @@ fn write_tlv_header<T: Write>(
     Ok(())
 }
 
+#[inline]
```

Comment:
I personally prefer the #[inline] to be after the comment just above the `fn` line to make it more obvious

Reply:
actually me too, i added the doc too fast and didn't see the #[inline]